src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -603,12 +603,17 @@ public with sharing class CRLP_RollupUI_SVC {
                         if (objectType == 'Rollup') {
                             Id rollupId = CRLP_Rollup_SEL.getRollupIdByDeveloperName(recordName);
                             response.completed = true;
-                            response.rollupItem = createRollupItem(CRLP_Rollup_SEL.getRollupById(rollupId));
+                            if (mode != 'delete') {
```

Comment:
Inequality operations should be `!==`

Reply:
as per quip convo, this one is apex so it's fine.